### PR TITLE
Ignore lines like `-e git://`

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by aplanas** [comment]
This is causing Liberty:Staging fail for manila-ui

**Comment by toabctl** [comment]
can you add a testcase, please?

**Comment by aplanas** [comment]
* Add a test case

File: tests.py [hunk 1]
```diff
@@ -83,12 +83,11 @@ def test_ignore_list(self):
                          pr.sanitize_requirements(
                              "hacking>=0.10.0,<0.11\nqpid-python"))
 
-    def test_with_markers(self):
```
Comment (toabctl) [inline tests.py]: Why are you removing this test?

**Comment by aplanas** [comment]
* Revert the replacement of a test.

**Comment by toabctl** [comment]
+1

**Comment by sayalilunkad** [comment]
LGTM +1

**Pull request merged**